src/__tests__/BaseBuilder.spec.js

```diff
@@ -482,4 +482,58 @@ describe('BaseBuilder', () => {
       });
     });
   });
+
+  describe('isEqual()', () => {
+    it('returns true with passed the same instance', () => {
+      const builder = new BaseBuilder();
+      expect(builder.isEqual(builder)).to.be.true;
+    });
+
+    it('can compare equivalent builders', () => {
+      const builder = new BaseBuilder();
+      expect(builder.isEqual(builder)).to.be.true;
```

Comment:
Remove this test–has same body as the one above it and same description as one below it